source/02_algo/exploration_rl_en.rst

```diff
@@ -0,0 +1,241 @@
+They experimentally verified that the forward prediction signal extracted in such a representation space can guide exploration well.
+In addition, there are also methods such as DIYAN [13]_, which is based on the objective function of mutual information to learn skill variables, which can automatically learn the distribution of state and skill by setting intrinsic rewards related to mutual information without external rewards. In subsequent hierarchical learning, imitation learning and exploration tasks.
+
+Memory-based exploration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Intrinsic reward-based exploration methods such as ICM and RND propose to measure the novelty of a state by predicting the error of the problem, providing a large intrinsic reward for a state with a large novelty and promoting exploration. These methods are difficult to explore under many sparse reward settings. Good results have been achieved on the task of , but there is a problem: \ **As the number of training steps of the agent increases, the prediction error of the prediction problem begins to decrease, and the exploration signal becomes smaller, that is, the agent is no longer encouraged to visit a certain Some states, but it is possible that these states are the states that must be visited to obtain extrinsic rewards**\ . And there may also be the following problems:
+
+-  The function approximation speed is relatively slow, and sometimes it cannot keep up with the speed of the agent's exploration, resulting in the intrinsic reward not well describing the novelty of the state.
+
+-  The reward for exploration is non-stationary.
+
+The exploration mechanism of storage-based exploration explicitly uses a Memory to maintain the historical state, and then gives the intrinsic reward value of the current state according to a certain metric between the current state and the historical state.
+
+Episodic Memory
+^^^^^^^^^^^^^^^^^
+
+NGU
+''''''''
+
+In order to solve the aforementioned problem of gradual attenuation of the exploration signal, the paper [5]_ `Never Give Up: Learning
+Directed Exploration
+Strategies <https://arxiv.org/abs/2002.06038>`__\  (Never
+Give Up)
+The agent adopts a new intrinsic reward generation mechanism that integrates the novelty of 2 dimensions: namely the life-long dimension of the inter-game intrinsic reward and the single-game dimension of the intra-game intrinsic reward , and also proposed to simultaneously learn a set of strategies with different degrees of exploration (directed
+exploratory policies) to collect more abundant samples for training. Among them, the intrinsic reward between games is maintained by maintaining an Episodic that stores the state of the game. Memory, calculated by calculating the distance between the current state and the k most similar samples in Memory. More details about NGU can be found in the blog [TODO].
```

Comment:
1. please put the same content in the same link
2. please capital the content within the brackets (directed exploratory policies)